views/partials/email-card.hbs

```diff
@@ -17,7 +17,7 @@
             ">
             <div class="e-toggle-info-wrapper">
               <div class="e-info">
-                <span class="e-address">{{ email }}</span>
+                <span class="e-address">{{{ email }}}</span>
```

Comment:
Do we need triple-stash here? This looks more like it could contain malicious markup, since it sometimes render an email address that hasn't been verified (and therefore sanitized?)